bigsi/bloom/bit_matrix_writer.py

```diff
@@ -0,0 +1,66 @@
+import os
+from bitarray import bitarray
+
+ROWS_PER_SLICE = 80  # must be divisible by 8. somewhere around 80 seems optimal
```

Comment:
This configuration is duplicated, I am wondering if we could have only one `ROWS_PER_SLICE` for `BitMatrixReader` and `BitMatrixWriter`

Reply:
No, they are not duplicated, although they look the same. This is for the number of rows holding in the memory before writing to the file. So this is mostly from the overall memory usage point of view, trying not to use too much memory.

So these two configuration items serve different purposes and could have different values, although they have the same name and same values at the moment.